script-library/docker-in-docker-debian.sh

```diff
@@ -380,14 +382,21 @@ dockerd_start="$(cat << 'INNEREOF'
     fi
     set -e
 
+    if [ -z "$DOCKER_DEFAULT_ADDRESS_POOL" ] || [ "$DOCKER_DEFAULT_ADDRESS_POOL" = "" ]
+    then
+        DEFAULT_ADDRESS_POOL=""
+    else
+        DEFAULT_ADDRESS_POOL="--default-address-pool $DOCKER_DEFAULT_ADDRESS_POOL"
+    fi
+
```

Comment:
This can be simplified to:

```suggestion
    if [ -n "$DOCKER_DEFAULT_ADDRESS_POOL" ]
    then
        DEFAULT_ADDRESS_POOL="--default-address-pool $DOCKER_DEFAULT_ADDRESS_POOL"
    fi
```

Reply:
Oops, nope. I missed the fact that you're defining a new, similarly-named variable, not altering the existing variable.  In this case, the available simplification is much more modest:

```suggestion
    if [ -z "$DOCKER_DEFAULT_ADDRESS_POOL" ]
    then
        DEFAULT_ADDRESS_POOL=""
    else
        DEFAULT_ADDRESS_POOL="--default-address-pool $DOCKER_DEFAULT_ADDRESS_POOL"
    fi

```